tests/appsec/contrib_appsec/test_fastapi.py

```diff
@@ -53,3 +53,7 @@ def headers(self, response):
 
     def body(self, response):
         return response.text
+
+    # TODO
+    # def location(self, response):
+    #     pass
```

Comment:
Old code left behind?

Reply:
no, it's a todo task. But it's failing anyway due to the fastapi issue I signaled before.